kitsune/users/models.py

```diff
@@ -69,6 +69,10 @@ class Profile(ModelBase, SearchMixin):
                             verbose_name=_lazy(u'City'))
     locale = LocaleField(default=settings.LANGUAGE_CODE,
                          verbose_name=_lazy(u'Preferred language'))
+    first_answer_email_sent = models.BooleanField(
+        default=False, verbose_name=_lazy('Has been sent a first answer contribution email.'))
+    first_l10n_email_sent = models.BooleanField(
+        default=False, verbose_name=_lazy('Has been sent a first revision contribution email.'))
```

Comment:
Actually, it kind of looks like you're using `verbose_name` in the way you'd use `help_text`.